app/gui/view/graph-editor/src/builtin/visualization/java_script/histogram.js

```diff
@@ -396,6 +395,17 @@ class Histogram extends Visualization {
         return { zoomElem, zoom, transformedScale }
     }
 
+    /** Removing `pointer-events` handling from brush element, as we want it to be inherited. D3 inserts
+     * `pointer-events: all` in the brush element and some of its children on brush creation and after brushing ends. */
```

Comment:
Is there any documentation to link, or is it just your discovery? You can add this little info to the comment.

Reply:
It's my discovery, but you are right, I'm adding a better description there